components/camel-properties/src/main/java/org/apache/camel/component/properties/PropertiesComponent.java

```diff
@@ -184,7 +187,8 @@ public Properties loadProperties() {
         }
 
         if (!sources.isEmpty()) {
-            for (PropertiesSource ps : sources) {
+            for (int i = sources.size(); i-- > 0; ) {
```

Comment:
Can you add a code commnet why we use revert ordering here

Reply:
done